giraffa-core/src/main/java/org/apache/giraffa/GiraffaPBHelper.java

```diff
@@ -24,18 +67,14 @@
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
-import org.apache.giraffa.GiraffaProtos.RenameStateProto;
-import org.apache.giraffa.GiraffaProtos.UnlocatedBlockProto;
-import org.apache.hadoop.hdfs.protocol.DatanodeInfo;
-import org.apache.hadoop.hdfs.protocol.proto.HdfsProtos.DatanodeInfosProto;
-import org.apache.hadoop.hdfs.protocolPB.PBHelper;
-import com.google.protobuf.ByteString;
 
 /**
  * Helper class, similar to PBHelper, for converting between Giraffa objects
  * and their protos. Also contains serialization/deserialization helpers.
  */
 public class GiraffaPBHelper {
+  private final static LocatedBlockProto LOCATED_BLOCK_PROTO = PBHelper.convert(
+      new LocatedBlock(new ExtendedBlock(), new DatanodeInfo[0]));
```

Comment:
Ah, thats why, because of how HBase works.
I created patch to fix that: https://issues.apache.org/jira/browse/HBASE-13646